creme/creme_core/static/creme_core/js/widgets/dialog/dialog.js

```diff
@@ -776,14 +776,22 @@ creme.dialogs = $.extend(creme.dialogs, {
 /*
  * Since jqueryui 1.10 some events are not allowed to get outside the popup.
  * So the custom Chosen widget will not work correctly (needs focus event on search field).
+ * Same issue with Select2
  */
-$.widget("ui.dialog", $.ui.dialog, {
-    _allowInteraction: function (event) {
-        if ($(event.target).closest(".chzn-drop").length) {
-            return true;
+if (!$.ui.dialog.prototype._allowInteractionPatched) {
+    $.ui.dialog.prototype._allowInteraction = function(e) {
+        if (typeof e !== "undefined") {
+            if ($(e.target).closest(".chzn-drop").length) {
+                return true;
+            }
```

Comment:
Add a comment like "DEPRECATED: to be removed when Chosen has been definitively removed in creme2.5" in order I do not forget it, please.

Reply:
np